contracts/ExoticPositionalMarkets/ExoticPositionalMarketManager.sol

```diff
@@ -140,10 +141,11 @@ contract ExoticPositionalMarketManager is Initializable, ProxyOwned, PausableUpg
             IThalesBonds(thalesBonds).sendCreatorBondToMarket(address(exoticMarket), msg.sender, fixedBondAmount);
             _activeMarkets.add(address(exoticMarket));
             exoticMarket.takeCreatorInitialPosition(_positionsOfCreator[0]);
-            emit MarketCreated(
```

Comment:
just small reminder if you change events, to change it in FE, if not we are cool :)

Reply:
Already implemented on FE to listen to new event